DBSCE/LocalKeyHelper-Mac.md

```diff
@@ -0,0 +1,96 @@
+<span style="color:red">!!! DRAFT - Please note this page is still in draft as we work on platform specific details
+</span>
+
+### Local key helper on Mac
+
+On macOS, a Local Key Helper is a protocol (`LocalKeyHelper.h`), and its implementation (`LocalKeyHelperImpl.h`/`LocalKeyHelperImpl.m`) is yet to be determined.
+
+The Local Key Helper will be implemented and shipped as an XPC service on macOS. It will be signed with app sandbox entitlement for securety requirement.
+
+To start the Local Key Helper XPC service, the service plist (property list) needs to be defined and registered in the launchAgent. This will ensure the service starts up when a idP sends a service connections.
```

Comment:
Let's add more details:

Each vendor will manage their own lifecycle of the XPC service depending on their needs and other functionality provided by the service. Each XPC service needs to be defined and registered with launchd as a launch agent. Vendors can decide whether to start XPC service on machine startup or on demand. Below is an example of an XPC registration. Please refer to Apple documentation for building and configuring XPC services.